### ISSUE TITLE
Preserve history?

### ISSUE THREAD

**Issue by wxmerkt** [exoeconomy/EXOS-Qt#2]
The history and origins of the code aren't preserved, i.e. it is impossible to trace fork history etc.

**Comment by turcol** [comment]
The last code is from Stratis Platform (https://github.com/stratisproject/stratisX) which is coming from Blackcoin, NovaCoin, PPCoin, up to Bitcoin
You can find the credit history in the Wallet under Help -> About CivX 
Copyright © 2009-2014 The Bitcoin developers
Copyright © 2011-2012 The PPCoin developers
Copyright © 2013-2014 The NovaCoin developers
Copyright © 2014-2015 The Blackcoin developers
Copyright © 2016-2017 The Stratis developers
Copyright © 2017-2018 The ExOFoundation developers

**Comment by wxmerkt** [comment]
Does it make sense to keep the commit history the same, i.e. fork and apply the current commits on top of it? This may help to later update/apply patches from previous forks or find and share security fixes/features etc.

**Comment by kentlangley** [comment]
Re-Opening

**Comment by kentlangley** [comment]
@wxmerkt You asked, "Does it make sense to keep the commit history the same, i.e. fork and apply the current commits on top of it?"

We chose a different path that suited our projects needs. Our choice to clone then work on our own version with appropriate attributions does not break our ability to accept code changes in the future or contribute upstream if we see fit.

What we know today about the upstream codebase we cloned suggests that we will not be accepting any changes from there. If you check, you'll see that it wasn't particularly active. On the flip side of that we don't expect to be submitting pull requests.

We will of course, monitor the upstream code and if needed, other similar Qt projects around the internet.

Hope that helps answer your questions.
